### ISSUE TITLE
Determine how best to set delay time without weird artifacts

### ISSUE THREAD

**Issue by trouscaillon** [chriswininger/FongPhone#17]

**Comment by chriswininger** [comment]
yeah, thought I discovered you can play those delay artifacts like a scratch pad and it makes awesome bubble sounds, so maybe bonus? I actually use it a lot that rather than as intended, but I also think that might be what causes my performances to degrade and the app to eventually crash; so, there's that too.
